.github/workflows/ci.yml

```
@@ -15,33 +15,24 @@ jobs:
     steps:
       - uses: actions/checkout@v4
-      - uses: pnpm/action-setup@v4
+      - uses: oven-sh/setup-bun@v2
         with:
-          version: 10.11.0
-      - uses: useblacksmith/setup-node@v5
-        with:
-          node-version: 23.11.1
-          cache: "pnpm"
+          bun-version: latest
       
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)